base/tps-client/src/processor/RA_Pin_Reset_Processor.cpp

```diff
@@ -404,7 +404,7 @@ TPS_PUBLIC RA_Status RA_Pin_Reset_Processor::Process(RA_Session *session, NameVa
         } 
 
         PR_snprintf((char *)configname, 256, "%s.%s.update.symmetricKeys.requiredVersion", OP_PREFIX, tokenType);
-        int v = RA::GetConfigStore()->GetConfigAsInt(configname, 0x00);
+        BYTE v = RA::GetConfigStore()->GetConfigAsInt(configname, 0x00);
         Buffer curKeyInfo = channel->GetKeyInfoData();
         BYTE nv[2] = { v, 0x01 };
```

Comment:
Could we defer the cast to this line instead, i.e.
```cpp
        int v = RA::GetConfigStore()->GetConfigAsInt(configname, 0x00);
        Buffer curKeyInfo = channel->GetKeyInfoData();
        BYTE nv[2] = { (BYTE) v, 0x01 };
```

Likewise for the other hunks?  This makes it more obvious that a cast is occurring,
rather than an implicit cast.